### ISSUE TITLE
Update 'Discover' view to match Android

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-iOS#201]

**Comment by n8fr8** [comment]
We might just put some place holder icons with a "coming soon" here.

**Comment by cstiens** [comment]
For this next release, we want a 'complete' look that is visually unified with Android, even if all of the features aren't included. Let's show placeholders like the 'sticker share' example in the image below. 

Here are the hex numbers for the text.
Title (ie. sticker share) - #808080
Subtext (Coming Soon) - #bfbfbf

@N-Pex I've attached a t-shirt icon to use for the theme. I checked font awesome and they don't have an equivalent.

I'm going to check with Kate about showing placeholders for photo stream and group chat as well. 

<img width="213" alt="screenshot 2016-12-12 10 41 04" src="https://cloud.githubusercontent.com/assets/1434597/21107009/c4fd5936-c055-11e6-8696-e0725b2b94eb.png">
